Sources/Utils/Data/Data+SecRng.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+import Foundation
+
+extension Data {
+    public static func secRngGenBytes(_ count: Int) -> Data {
+        Data(withFixedLengthMcMutableBufferInfallible: count) {
+            SecRandomCopyBytes(kSecRandomDefault, count, $0.pointee.buffer) == errSecSuccess
```

Comment:
nit: the error printed here would say `"Infallible LibMobileCoin function failed."` which is a little misleading because its not coming from a LibMobileCoin FFI call. 

Maybe make a new one with a different error message, or change the existing to be more generic like`"Infallible Unsafe Mutable Data Allocation failed."`